packages/config/src/projects/(tokens)/yieldfi/config.jsonc

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "../../../../discovery/schemas/config.v2.schema.json",
+  "name": "yieldfi",
+  "import": ["../../globalConfig.jsonc"],
+  "maxDepth": 2,
+  "initialAddresses": [
+    "eth:0x2e3c5e514eef46727de1fe44618027a9b70d92fc",
+    "eth:0x19ebd191f7a24ece672ba13a302212b5ef7f35cb",
+    "eth:0x10472f91f67C49260BDD65F016ea0757bb1Fc91e",
```

Comment:
Why are these administrators in initialAddresses? What if they stop being administrators. Let's rely on discovery to find them. Initial addresses should be kept as minimal as possible.